lib/linux_kernel/kcookie.c

```diff
@@ -26,11 +26,26 @@
  */
 #include <linux/uaccess.h> /* probe_kernel_address */
```

Comment:
Can we remove this header file?

Reply:
No. pagefault_enable()/disable() are declared there. But we can remove mentioning probe_kernel_address at it - yes, good catch.